### PR TITLE
[17.0][MIG] product_set: Migration to 17.0

### PR DESCRIPTION
**Pull request by NICO-SOLUTIONS**
standard migration to 17.0

### CODE REVIEW

**Review by cormaza — APPROVED** [review]
LGTM

File: product_set/tests/test_product_set.py [hunk 1]
```diff
@@ -10,9 +10,6 @@ class TestProductSet(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
Comment (rousseldenis) [inline product_set/tests/test_product_set.py]: @NICO-SOLUTIONS Why removing those lines ?

File: product_set/models/product_set.py [hunk 2]
```diff
@@ -32,14 +32,20 @@ class ProductSet(models.Model):
         "it's going to be available for all of them.",
     )
 
+    display_name = fields.Char(
+        compute="_compute_display_name", string="Display Name", store=True
+    )
+
     def name_get(self):
-        return [(rec.id, rec._name_get()) for rec in self]
+        return [(rec.id, rec.display_name) for rec in self]
```
Comment (rousseldenis) [inline product_set/models/product_set.py]: If you implement a display_name, you don't need to override name_get() (moreover deprecated in 17.0).

See : https://github.com/odoo/odoo/blob/17.0/odoo/models.py#L1676

File: product_set/views/product_set.xml [hunk 3]
```diff
@@ -24,7 +24,7 @@
                         name="web_ribbon"
                         title="Archived"
                         bg_color="bg-danger"
-                        attrs="{'invisible': [('active', '=', True)]}"
+                        invisible="active == 'True'"
```
Comment (rousseldenis) [inline product_set/views/product_set.xml]: You can use 'active' only

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: product_set/views/product_set.xml (hunk 3 above)
Comment (NICO-SOLUTIONS) [reply product_set/views/product_set.xml]: thx

File: product_set/models/product_set.py (hunk 2 above)
Comment (NICO-SOLUTIONS) [reply product_set/models/product_set.py]: true... thx

File: product_set/tests/test_product_set.py (hunk 1 above)
Comment (NICO-SOLUTIONS) [reply product_set/tests/test_product_set.py]: there is no dependency to sale module in manifest. This test will be done in sale_product_set. So i guess this i not necessary here?!

**Comment by MohamedOsman7** [comment]
Code LGTM (however the functionality cannot be tested because of the runboat)

**Comment by NICO-SOLUTIONS** [comment]
@MohamedOsman7 
now you can check th erunboat

**Review by MohamedOsman7 — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
This has been improved recently in v16, and it's worth to include that changes: #1548

**Comment by NICO-SOLUTIONS** [comment]
@pedrobaeza  
I will take care of it soon. Thanks for the information

**Comment by NICO-SOLUTIONS** [comment]
super seeded by PR1554

**Pull request closed without merging**